src/js/components/RadioButton/stories/Disabled.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 
-import { Box, RadioButton } from 'grommet';
+import { Box, Grommet, RadioButton } from 'grommet';
 
 export const Disabled = () => (
-  <Box align="center" pad="large" gap="large">
-    <RadioButton
-      label="option 1"
-      name="name"
-      value="option 1"
-      checked
-      disabled
-    />
-  </Box>
+  <Grommet
```

Comment:
I think there are enough stories demoing this behavior, can we leave this story as it was?